baus/subsidies.py

```diff
@@ -248,27 +181,17 @@ def inclusionary_housing_revenue_reduction(feasibility, units):
     households = orca.get_table("households")
     buildings = orca.get_table("buildings")
     parcels_geography = orca.get_table("parcels_geography")
-    policy = orca.get_injectable("policy")
 
-    if orca.get_injectable("scenario") in policy["inclusionary_d_b_enable"]:
-        h = orca.merge_tables("households",
-                              [households, buildings, parcels_geography],
-                              columns=["juris_name",
-                                       "income",
-                                       "pba50chcat"])
-        AMI = h.groupby(h.pba50chcat).income.quantile(.5)
-    elif orca.get_injectable("scenario") in policy["inclusionary_fb_enable"]:
-        h = orca.merge_tables("households",
-                              [households, buildings, parcels_geography],
-                              columns=["juris_name",
-                                       "income",
-                                       "fbpchcat"])
-        AMI = h.groupby(h.fbpchcat).income.quantile(.5)
-    else:
-        h = orca.merge_tables("households",
-                              [households, buildings, parcels_geography],
-                              columns=["juris_name", "income"])
-        AMI = h.groupby(h.juris_name).income.quantile(.5)
+    run_setup = orca.get_injectable("run_setup")
+    policy = orca.get_injectable("policy")
```

Comment:
See discussion in the `earthquake.py` review comment. I haven't experienced issues with alternative between these in the past, but it might be worth setting standards that we like.